File: Random Marathon/week 76/G 28038.c
#include <stdio.h>
#define _USE_MATH_DEFINES
#include <math.h>

int main(void)
{
    int T;
    scanf("%d", &T);

    while (T--)
    {
        int c;
        scanf("%d", &c);

        double a, b;
        scanf("%lf %lf", &a, &b);

        if (c == 1)
        {
            double r = sqrt(pow(a, 2) + pow(b, 2));
            double t = atan2(b, a);
            if (t < 0)
            {
                t += M_PI * 2;
            }
            printf("%.6lf %.6lf\n", r, t);
        }

        if (c == 2)
        {
            double x = a * cos(b);
            double y = a * sin(b);

            printf("%.6lf %.6lf\n", x, y);
        }
    }

    return 0;
}

File: Random Marathon/week 76/G 28038.py
import math


def main():

    T = int(input())

    for _ in range(T):

        c = int(input())

        a, b = map(float, input().split())

        if c == 1:
            r = (a**2 + b**2) ** 0.5
            t = math.atan2(b, a)
            if t < 0:
                t += math.pi * 2
            print(f"{r:.7f} {t:.7f}")

        if c == 2:
            x = a * math.cos(b)
            y = a * math.sin(b)
            print(f"{x:.7f} {y:.7f}")


main()
